pkgs/by-name/pr/presenterm/package.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "presenterm";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mfontanini";
+    repo = pname;
+    rev = version;
+    hash = "sha256-mNWnUUezKIffh5gMgMMdvApNZZTxxB8XrL0jFLyBxuk=";
+  };
+
+  cargoHash = "sha256-JLPJLhWN/yXpPIHa+FJ2aQ/GDUFKtZ7t+/8rvR8WNKM=";
+
+  meta = with lib; {
+    description = "A terminal based slideshow tool.";
+    homepage = "https://github.com/mfontanini/presenterm";
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ mikaelfangel ];
+    mainProgram = pname;
```

Comment:
```suggestion
    mainProgram = "presenterm";
```

Reply:
Not sure on why this shouldn't be pname. Can you elaborate on the suggestion? (:

Reply:
this is bad practice, they are semantically different things, same w repo vs pname.

Reply:
Hmm, I see. I'll change it then, but this style is widely adopted throughout the repo.

A quick search in the repo gives 5.1k code results. [See here](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+%22repo+%3D+pname%3B%22&type=code)